### PR TITLE
Amal1

### PR DESCRIPTION
**Pull request by Amal031**
Hello0o0o guys !

This is our design animated signup / registration, login / signing & r/ forgot password form using HTML, CSS 
but still we didn't make it responsive for all devices

waiting for your reviews!!

Hussin ! Remember new week is near ;! !!

Regrads,
team 1

### CODE REVIEW

File: pages/signin.html [hunk 1]
```diff
@@ -5,11 +5,53 @@
...
 </head>
 
 <body>
+    <header>
+        <a class="logo" href="#">my<span>Bank</span></a>
 
+    </header>
+    <main>
+        <div class="mainSvg">
+            <h1> We Keep Customerers <span>Coming Back For More !</span> </h1>
```
Comment (HosenNaser) [inline pages/signin.html:20]:
```suggestion
            <span> We Keep Customerers <h1>Coming Back For More !</h1> </span>
```
I prefer to put the h1 inside the span not the other way

**Review by HosenNaser — CHANGES_REQUESTED** [review]
check my suggesting

File: pages/signin.html [hunk 2]
```diff
@@ -5,11 +5,53 @@
...
+                    <input type="email" placeholder="Email">
+                </div>
+                <div class="inputDiv">
+                        <label>Password</label>
+                        <i class="bi bi-lock-fill"></i>
+                        <input type="password" placeholder="Password">               
+                </div>
+
+                <div class="btnDiv">
+                    <p class="special_txt">Forgot password</p>
```
Comment (HosenNaser) [inline pages/signin.html:40]: change it to a link

**Review by HosenNaser — CHANGES_REQUESTED** [review]
change the forgot password to a link

File: pages/signin.html (hunk 2 above)
Comment (Amal031) [reply pages/signin.html]: hey Hussin ,Thanks for you're commit
though in the beginning  I thought this is  really not matter if we match with the right "id" p-element. I fixed it.

**Review by HosenNaser — APPROVED** [review]

**Review by HosenNaser — APPROVED** [review]

**Pull request merged**